src/crimson/os/seastore/onode.h

```diff
@@ -54,19 +54,25 @@ class Onode : public boost::intrusive_ref_counter<
 {
 protected:
   virtual laddr_t get_hint() const = 0;
+  const uint32_t default_data_reservation = 0;
```

Comment:
Seems there is no user of this value.

Reply:
Yeah, looks like SeaStore passes it into ObjectDataHandler directly.

Reply:
I think we're just blocked on this one.

Reply:
@athanatos @cyx1231st Done:-)